airflow/ui/src/pages/DagsList/DagsList.tsx

```diff
@@ -118,9 +118,24 @@ const cardDef: CardDef<DAGResponse> = {
   },
 };
 
+const DAGS_LIST_DISPLAY = "dags_list_display";
+
 export const DagsList = () => {
   const [searchParams, setSearchParams] = useSearchParams();
-  const [display, setDisplay] = useState<"card" | "table">("card");
+  const [display, setDisplay] = useState<"card" | "table">(() => {
+    const storage = localStorage.getItem(DAGS_LIST_DISPLAY);
+
+    if (storage === "card" || storage === "table") {
+      return storage;
+    }
+
+    return "card";
+  });
```

Comment:
Maybe we should extract that to a service class, or some utility custom hook with some kind of `useReducer` in it.

The idea is to avoid to have each components interacting with the storage in their own way. (types, names, etc...) Only 1 class / utility code should do that and guarantee its integrity, having the storage full typing. Will also limit having storage code all over the place.

Reply:
Moved local storage over to https://usehooks-ts.com/react-hook/use-local-storage